src/features/ModelPlan/Collaborators/AddCollaborator/index.tsx

```diff
@@ -357,8 +357,12 @@ const Collaborators = () => {
                             teamRolesConfig.options,
                             undefined,
                             isModelLead && isLastModelLead(allCollaborators)
-                              ? TeamRole.MODEL_LEAD
-                              : ''
+                              ? {
+                                  [TeamRole.MODEL_LEAD]: collaboratorsT(
+                                    `teamRoles.options.${TeamRole.MODEL_LEAD}`
+                                  )
+                                }
+                              : undefined
```

Comment:
This has nothing to do with my code, but making these changes to address type errors